lib/Models/GeoJsonCatalogItem.js

```diff
@@ -408,20 +401,16 @@ function loadGeoJson(geoJsonItem) {
 
     */
 
-    function defaultColor(colorString, palette) {
+    function defaultColor(colorString, name) {
         if (colorString === undefined) {
-            return getRandomColor(palette, geoJsonItem.name);
+            var color = Color.fromCssColorString(getRandomCssColor(standardCssColors.highContrast, name));
+            color.alpha = 0.6;
```

Comment:
We probably want points to be opaque, right?  Code below gives polygons (via the fill property) an alpha of 0.75, and that could maybe default to a bit lower (like 0.6).